projects/js-packages/connection/components/disconnect-card/index.jsx

```diff
@@ -3,6 +3,7 @@
  */
 import React from 'react';
 import PropTypes from 'prop-types';
+import { numberFormat } from 'components/number-format';
```

Comment:
This is not going to work I'm afraid, since `components/number-format` is not part of the Connection package. Maybe we should move that component to `@automattic/jetpack-components` so it can be used elsewhere?

Reply:
Excellent catch @jeherve - thanks! I like the idea of moving the number-format utility into `@automattic/jetpack-components`. I'll take a look at that in a separate PR for clarity and then rebase this one.

Reply:
Rebased against a new branch that does some refactoring for numberFormat, see #22215 for the base branch PR.